Flick/Group/GroupResultViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  GroupResultViewController.swift
+//  Flick
+//
+//  Created by Haiying W on 1/25/21.
+//  Copyright © 2021 flick. All rights reserved.
+//
+
+import UIKit
+
+class GroupResultViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let resultsTableView = UITableView()
+    private let votingStatusImageView = UIImageView()
+    private let votingStatusLabel = UILabel()
+
+    // MARK: - Private Data Vars
```

Comment:
can we start to follow new style of having the _reuseIdenfier_ as a static variable in the class file for the cell?